packages/react-search-ui/src/containers/__tests__/SearchBox.test.js

```diff
@@ -420,6 +479,25 @@ describe("onSelectAutocomplete", () => {
     expect(passedAutocompleteResults).toBeDefined();
     expect(passedDefaultOnSelectAutocomplete).toBeDefined();
   });
+
+  it("will call setSearchTerm if no onSelectAutocomplete is specified and a suggestion is selected", () => {
```

Comment:
Is this one related to the new clearFilters logic or just something you happened to notice along the way? I think it's great either way, but my one comment for the file overall is that it's getting pretty long and hard to reason about, so I might suggest either breaking up visually with `/**` comment docblocks or grouping all tests with `describe()` parents (there's a couple loose tests that are kind of floating around). That may be something for another day/as tech debt though :)

Reply:
@constancecchen It's one that I noticed was missing. And I agree, as I was writing this I was pretty overwhelmed. If I can find the time I will clean it up, if not, hopefully it can be addressed in a subsequent PR!